nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Discover/DiscoveryWorkerTests.GlobalJson.cs

```diff
@@ -50,7 +84,7 @@ await TestDiscoveryAsync(
                     ("global.json", """
                         {
                           "sdk": {
-                            "version": "2.2.104",
+                            "version": "2.2.104", INVALID JSON
```

Comment:
Previous `ReportsFailure()` test used a trailing comma to throw an error. This PR allows that behavior so I added text to make the JSON invalid. These changes were tested by doing a run against this repo: https://github.com/sebasgomez238/dpdbot-global-json